lib/usdUfe/ufe/Utils.cpp

```diff
@@ -1330,6 +1358,18 @@ std::string defaultMaterialScopeName()
                                           : kDefaultMaterialScopeName;
 }
 
+void setTransform3dMatrixOpNameFn(Transform3dMatrixOpNameFn fn)
+{
+    // This function is allowed to be null in which case there is
+    // no special transform3d matrix op name.
+    gTransform3dMatrixOpNameFn = fn;
+}
+
+const char* getTransform3dMatrixOpName()
+{
+    return gTransform3dMatrixOpNameFn ? gTransform3dMatrixOpNameFn() : nullptr;
```

Comment:
We override this one (from MayaUsd) to provide the value from the env var. The default implementation here is to just return nullptr (which is checked for where this is called).